docs/docs/configuration/auth.md

```diff
@@ -70,6 +80,9 @@ https://www.googleapis.com/auth/admin.directory.user.readonly
 
 Note: The user is checked against the group members list on initial authentication and every time the token is refreshed ( about once an hour ).
 
+#### 
+
+
```

Comment:
please remove these 3 lines

Reply:
Thanks! I should've caught that myself 😞